charts/gateway/templates/node-properties-secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if not .Values.disklessConfig  }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name:  {{ template "nodePropertiesSecretName" . }}
+  annotations:
+    description: Template for Secrets for Gateway node.properties
+  labels:
+    app: {{ template "gateway.name" . }}
+    chart: {{ template "gateway.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+  node.properties: {{ .Files.Get "node.properties" | b64enc }}
```

Comment:
Where does user need to put this node.properties so this template can pick it up?

Reply:
Updated node-properties-secret , now we can upload node.properties similar to license
Example:

`helm -n dk670466-gw upgrade -i dk670466-gw --set-file "license.value=license.xml" --set "license.accept=true" --set-file "disklessConfig.value=node.properties"  -f gateway-db-diskless-values.yaml  .`